client/my-sites/activity/filterbar/date-range-selector.jsx

```diff
@@ -275,7 +275,7 @@ export class DateRangeSelector extends Component {
 							onClick={ props.onTriggerClick }
 							ref={ props.buttonRef }
 						>
-							{ this.getFormattedDate( from, to ) }
+							{ customLabel ? customLabel : this.getFormattedDate( from, to ) }
```

Comment:
Can we just use label? What makes this label custom?

Reply:
We could just use label, but I don't want future users of this component to feel compelled to provide a label. The name of this prop feels appropriate because it's optional.